Source/JavaScriptCore/wasm/WasmIPIntGenerator.cpp

```diff
@@ -2764,6 +2778,10 @@ void IPIntGenerator::addTailCallCommonData(const FunctionSignature& signature)
         data += 1;
         mINTBytecode.removeLast();
     }
+
+    uint32_t numStackValues = WTF::roundUpToMultipleOf(stackAlignmentRegisters(), wasmCallingConvention().numberOfStackValues(signature));
+    ASSERT(!(numStackValues % 2));
+    m_metadata->appendMetadata(numStackValues);
```

Comment:
Don't we need to update the max frame size by this amount since we copy the frame right? Would be worth adding a test for this.